content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+description: >
+  Learn about the Kubernetes ServiceAccount object.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts? {#what-are-service-accounts}
+
+A service account is a type of non-human account that, in Kubernetes, provides
+a distinct identity in a Kubernetes cluster. Application Pods, system
+components, and entities inside and outside the cluster can use a specific
+ServiceAccount's credentials to identify as that ServiceAccount. This identity
+is useful in various situations, including authenticating to the API server or
+implementing identity-based security policies.
+
+Service accounts exist as ServiceAccount objects in the API server. Service
+accounts have the following properties:
+
+*  **Namespaced:** Each service account is bound to a Kubernetes
+   {{<glossary_tooltip text="namespace" term_id="namespace">}}. Every namespace
+   gets a [`default` ServiceAccount](#default-service-account) upon creation.
+
+*  **Lightweight:** Service accounts exist in the cluster and use the structure
+   defined in the Kubernetes API. You can quickly create service accounts to
+   enable specific tasks.
+
+*  **Portable:** A configuration bundle for a complex containerized workload
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. By default, user accounts don't exist in the Kubernetes API server; instead, the API server treats user identities as opaque
+data. You can authenticate as a user account using multiple methods. Some
+Kubernetes distributions might add custom extension APIs to represent user
+accounts in the API server.
+
+{{< table caption="Comparison between service accounts and users" >}}
+
+| Description | ServiceAccount | User or group |
+| --- | --- | --- |
+| Location | Kubernetes API (ServiceAccount object) | External |
+| Scope | Namespace | Cluster |
+| Access control | Kubernetes RBAC or other [authorization mechanisms](/docs/reference/access-authn-authz/authorization/#authorization-modules) | Kubernetes RBAC or other identity and access management mechanisms |
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts {#default-service-accounts}
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions by default other than the
+[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
```

Comment:
We should try to be slightly neutral about how `system:authenticated` gets permissions (it could be RBAC, it could be another mechanism).

Reply:
Since RBAC is the "built-in" mechanism, I don't have an issue with changing the link to the suggestion. @sftim let me know which you think is better for here

Reply:
RBAC is in-tree but is not part of what we consider as _certified Kubernetes_.

Reply:
```suggestion
[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
```

that Kubernetes grants to all authenticated accounts using an access control
mechanism such as role-based access control (RBAC).

Reply:
Implemented Tim's with minor tweaks